wgpu/backends/wgpu_native/_api.py

```diff
@@ -2735,6 +2724,66 @@ def begin_render_pass(
         encoder._objects_to_keep_alive = objects_to_keep_alive
         return encoder
 
+    # Pulled out from create_render_pass because it was too large.
+    @staticmethod
+    def _create_c_render_pass_depth_stencil_attachment(ds_attachment):
+        view = ds_attachment["view"]
+        depth_read_only = stencil_read_only = False
+        depth_load_op = depth_store_op = stencil_load_op = stencil_store_op = 0
+        depth_clear_value = stencil_clear_value = 0
+        depth_keys_okay = stencil_keys_okay = False
+        # All depth texture formats have "depth" in their name.
+        if "depth" in view.texture.format:
+            depth_read_only = ds_attachment.get("depth_read_only", False)
+            if not depth_read_only:
+                depth_keys_okay = True
+                depth_load_op = ds_attachment["depth_load_op"]
+                depth_store_op = ds_attachment["depth_store_op"]
+                if depth_load_op == "clear":
```

Comment:
Is there a better way of doing this?  Will anyone specify "clear" by its integer value?